harness/determined/common/experimental/experiment.py

```diff
@@ -274,6 +280,25 @@ def unarchive(self) -> None:
         bindings.post_UnarchiveExperiment(self._session, id=self._id)
         self.archived = False
 
+    def move_to_project(self, workspace_name: str, project_name: str) -> None:
+        """Move an experiment to a different project.
+
+        Updates both the local object and the master database with the new project and workspace.
+
+        Args:
+            project_name: The name of the project to move the experiment to.
+            workspace_name: The name of the workspace with the project.
+        """
```

Comment:
Args:
        project_name: The name of the destination project for the experiment.
        workspace_name: The name of the workspace containing the project.
    """